generators/chipyard/src/main/scala/IOBinders.scala

```diff
@@ -232,21 +261,38 @@ class WithSimNIC extends OverrideIOBinder({
 // accessible to the IOBinder
 // DOC include start: WithSimAXIMem
 class WithSimAXIMem extends OverrideIOBinder({
-  (system: CanHaveMasterAXI4MemPort with BaseSubsystem) => SimAXIMem.connectMem(system)(system.p); Nil
```

Comment:
Are we going to remove the helper function / object?

Reply:
Which one? The AddIOCells function?

Reply:
Nevermind, I forgot that this was present in RC (not in TCIP)